File: Benchmarks/enterprise/python/flask/testcode/benchmark_test_52040.py
# SPDX-License-Identifier: Apache-2.0
from flask import request, jsonify


def BenchmarkTest52040():
    upload_name = request.files['upload'].filename
    data = '{}'.format(upload_name)
    eval(str(data))
    return jsonify({"result": "success"})

File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest52040.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest52040 {

    @GetMapping("/BenchmarkTest52040/{pathId}")
    public void BenchmarkTest52040(@PathVariable("pathId") String pathId, HttpServletRequest request, HttpServletResponse response) throws Exception {
        String pathValue = pathId;
        java.text.MessageFormat fmt = new java.text.MessageFormat("payload={0}");
        String data = fmt.format(new Object[]{pathValue});
        Integer.parseInt(data);
        response.setContentType("application/json");
        response.getWriter().print("{\"id\":0}");
    }
}
